src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx

```diff
@@ -514,7 +516,7 @@ function ReportActionCompose({
                                             isScrollLikelyLayoutTriggered={isScrollLikelyLayoutTriggered}
                                             raiseIsScrollLikelyLayoutTriggered={raiseIsScrollLikelyLayoutTriggered}
                                             reportID={reportID}
-                                            policyID={report?.policyID ?? '-1'}
+                                            policyID={report?.policyID}
```

Comment:
policyID accepts a string, so we can't do that unless we convert the number to a string.